### ISSUE TITLE
Unlocking Specific Achievement

### ISSUE THREAD

**Issue by AlphaDoe** [YoYoGames/GMEXT-EpicOnlineServices#5]
Hi, I am unable to find the command to set and unlock an achievement.

For example, the Steamworks extension has a function called "steam_set_achievement("Achievement ID") that I can code in gamemaker to unlock specific achievement.

However, I went through all the Functions of the EpicOnlineService extension. The only function that mentioned about unlock achievement is "EpicGames_Achievements_UnlockAchievement(userID,AchievementId)"

I replaced the AchievementId with the "Achievement ID" configured in my Epic developer portal, as shown below and it doesn't work.
"EpicGames_Achievements_UnlockAchievement(userID,“Intermediate Survivor”);"

Is this an issue? If not how do I code in gamemaker to unlock the above using the EpicOnlineService extension? An example line will be good, thanks.

**Comment by DiasFranciscoA** [comment]
okay the AchievementID you need to use is the Achievement Name (which seems to be the one you are already using)

<img width="950" alt="image" src="https://github.com/YoYoGames/GMEXT-EpicOnlineServices/assets/85873067/374ff11b-9b0d-4a80-ab02-d77f037f3145">

this is working correctly on our side. On the demo, if you replace your credentials on the demo project and go into the achievements tab it should query all your achievements. If you click the icon it should unlock them, can you check if that is working for you?

**Comment by AlphaDoe** [comment]
Dear Francisco,

I am pleased to inform you that the issue has been resolved successfully. It appears that the problem was likely caused by a bug within Gamemaker.

Upon further investigation, I discovered that copying and pasting the Achievement ID "Intermediate Survivor" directly from the Epic Dev portal into the Gamemaker code resulted in a syntax error. However, when I manually typed the exact same ID, it worked without any issues. This inconsistency suggests that there may be a bug within Gamemaker affecting the copying and pasting functionality.

Nevertheless, I am glad to report that I was able to overcome this obstacle and resolve the issue. Therefore, I kindly request you to close the case. I appreciate your assistance and the time you dedicated to addressing this matter.

Thank you once again.
